setup.py

```diff
@@ -16,16 +16,6 @@
 import sys
 import warnings
 
-if __name__ == '__main__' and sys.argv[1] == "gevent_nosetests":
-    print("Running gevent tests")
-    from gevent.monkey import patch_all
-    patch_all()
-
-if __name__ == '__main__' and sys.argv[1] == "eventlet_nosetests":
-    print("Running eventlet tests")
-    from eventlet import monkey_patch
-    monkey_patch()
-
```

Comment:
This monkey patch in the gevent/eventlet case is now handled [in test infrastructure](https://github.com/datastax/python-driver/blob/3.29.1/tests/__init__.py#L67-L84) so we shouldn't need to support it as a CLI arg as well.